andela/ah-backend-thor
# Ch integrate coveralls 161382386

What does this PR do?
Intergrates coveralls

Description of Task to be completed?
Intergrate Coveralls which is a code coverage service

How should this be manually tested?
On the repository on the readme click on the coveralls badge this will redirect to the coveralls web page and there you should see more details on the coverage

Any background context you want to provide?
This is to ensure we are properly informed just how well our code base is tested which in turn provides quality assurance for the product

What are the relevant pivotal tracker stories?
#161382386
